cmd/up/controlplane/get.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2022 Upbound Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controlplane
+
+import (
+	"context"
+
+	"github.com/alecthomas/kong"
+	"github.com/pterm/pterm"
+
+	cp "github.com/upbound/up-sdk-go/service/controlplanes"
+
+	"github.com/upbound/up/internal/upbound"
+)
+
+// AfterApply sets default values in command after assignment and validation.
+func (c *getCmd) AfterApply(kongCtx *kong.Context, upCtx *upbound.Context) error {
+	kongCtx.Bind(pterm.DefaultTable.WithWriter(kongCtx.Stdout).WithSeparator("   "))
+	return nil
+}
+
+// listCmd list control planes in an account on Upbound.
+type getCmd struct {
+	Name string `arg:"" required:"" help:"Name of control plane."`
+}
+
+// Run executes the get command.
+func (c *getCmd) Run(p pterm.TextPrinter, pt *pterm.TablePrinter, cc *cp.Client, upCtx *upbound.Context) error {
+	ctp, err := cc.Get(context.Background(), upCtx.Account, c.Name)
+	if err != nil {
+		return err
+	}
+
+	// We convert to a list so we can match the output of the list command
+	cpList := cp.ControlPlaneListResponse{
+		ControlPlanes: make([]cp.ControlPlaneResponse, 1),
+	}
+	cpList.ControlPlanes[0] = *ctp
+	return PrintControlPlanes(&cpList, pt)
```

Comment:
We can construct inline here:
```suggestion
	cpList := cp.ControlPlaneListResponse{
		ControlPlanes: []cp.ControlPlaneResponse{*ctp},
	}
	return PrintControlPlanes(&cpList, pt)
```